tests/backup/backup_test.go

```diff
@@ -3117,6 +3118,179 @@ var _ = Describe("{BackupLocationWithEncryptionKey}", func() {
+				BackupLocationUID = uuid.New()
+				BackupLocationMap[BackupLocationUID] = backupLocation
+				CreateBackupLocation(provider, backupLocation, BackupLocationUID, CredName, CloudCredUID,
+					bucketName, orgID, "")
+			}
+		})
+
+		Step("Register cluster for backup", func() {
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			CreateSourceAndDestClusters(orgID, "", "", ctx)
+			clusterStatus, clusterUid = Inst().Backup.RegisterBackupCluster(orgID, SourceClusterName, "")
+			dash.VerifyFatal(clusterStatus, api.ClusterInfo_StatusInfo_Online, "Verifying backup cluster")
+		})
+
+		Step("Start backup of application to bucket", func() {
+			for _, namespace := range bkpNamespaces {
+				ctx, err := backup.GetAdminCtxFromSecret()
+				dash.VerifyFatal(err, nil, "Getting context")
+				preRuleUid, _ := Inst().Backup.GetRuleUid(orgID, ctx, preRuleNameList[0])
+				postRuleUid, _ := Inst().Backup.GetRuleUid(orgID, ctx, postRuleNameList[0])
+				backupName := fmt.Sprintf("%s-%s-%s", BackupNamePrefix, namespace, backupLocationName)
+				CreateBackup(backupName, SourceClusterName, backupLocation, BackupLocationUID, []string{namespace},
+					labelSelectors, orgID, clusterUid, preRuleNameList[0], preRuleUid, postRuleNameList[0], postRuleUid, ctx)
+			}
+		})
+
+		Step("Restoring the backed up application", func() {
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			for _, namespace := range bkpNamespaces {
+				backupName = fmt.Sprintf("%s-%s", BackupNamePrefix, namespace)
+				restoreName = fmt.Sprintf("%s-%s", "test-restore", namespace)
+				CreateRestore(restoreName, backupName, namespaceMapping, destinationClusterName, orgID, ctx)
+			}
+		})
+
+		Step("Resize volume after the restore is completed", func() {
+			log.InfoD("Resize volume after the restore is completed")
+			var err error
+			for _, ctx := range contexts {
+				var appVolumes []*volume.Volume
+		    log.InfoD(fmt.Sprintf("get volumes for %s app", ctx.App.Key))
+		    appVolumes, err = Inst().S.GetVolumes(ctx)
+		    log.FailOnError(err, "Failed to get volumes for app %s", ctx.App.Key)
+		    dash.VerifyFatal(len(appVolumes) > 0, true, "App volumes exist?")
+				var requestedVols []*volume.Volume
+				log.InfoD(fmt.Sprintf("Increase volume size %s on app %s's volumes: %v",
+					Inst().V.String(), ctx.App.Key, appVolumes))
+					requestedVols, err = Inst().S.ResizeVolume(ctx, Inst().ConfigMap)
+				log.FailOnError(err, "Volume resize successful ?")
+				log.InfoD(fmt.Sprintf("validate successful volume size increase on app %s's volumes: %v",
+					ctx.App.Key, appVolumes))
+					for _, v := range requestedVols {
+						// Need to pass token before validating volume
+			      params := make(map[string]string)
+						if Inst().ConfigMap != "" {
+			        params["auth-token"], err = Inst().S.GetTokenFromConfigMap(Inst().ConfigMap)
```

Comment:
Where are we configuring `Inst().ConfigMap` in the backup cluster?

Reply:
Not exactly sure @lsrinivas-pure can you please shed some light here?

Reply:
it is configured in common.go where are parsing flags. you can pass "" as you dont need to any security annotations to be updated.